game.h

```diff
@@ -15,6 +15,9 @@ class Game {
   // This function does not return because the game loop is inside.
   bool Run();
 
+  // Limits the frame rate.
+  void LimitFrames();
```

Comment:
heads up, when a review comment starts with the word "nit:", it just means, "Maybe I'm being nit-picky, but I'd prefer this...". 

On that note:
nit: How bout "LimitFrameRate"?